girder-dandi-archive/setup.cfg

```diff
@@ -1,2 +1,6 @@
 [bdist_wheel]
 universal = 1
+
+[flake8]
```

Comment:
`setup.cfg` is on the way out in the Python ecosystem. I'd put this config block in `tox.ini`.